### PR TITLE
Configuring the PHP CS Fixer tool.

### PR DESCRIPTION
**Pull request by weerd**
### What's this PR do?

This pull request adds PHP CS Fixer tool to help format/lint our PHP. It uses a [new ruleset](https://github.com/weerd/php-style/blob/master/src/rules/laravel-prettier.php) based on the laravel ruleset but modified to account for differences with Prettier to avoid conflicts in formatting. If you do check the formatting changes, it looks like it was mostly import statement ordering fixes and docblock fixes.

### How should this be reviewed?

Do not review via the "Files changed" tab or you might have faint; a number of PHP files got formatted.

You mostly just need to check out the following commits:

- [Initial configuration of the PHP CS Fixer tool](https://github.com/DoSomething/aurora/pull/259/commits/c8e23c789137973db936f8bc8b48a25ba6f9cdb7)
- [Using updated php-style package.](https://github.com/DoSomething/aurora/pull/259/commits/3082140eb06e4590e5bacac7bb8679bdb67cd980)
- [Configuring the PHP CS Fixer tool.](https://github.com/DoSomething/aurora/pull/259/commits/6646b064e85521bcfc2566af3ae64c5428777946)

The tool is run via `composer` (typically run within your Homestead). It can be run using the `composer format` command.

### Any background context you want to provide?

This relates to [RFC #11 for PHP Code Formatting](https://github.com/DoSomething/rfcs/blob/master/011-php-code-formatting.md). In a [previous PR](https://github.com/DoSomething/aurora/pull/258), Prettier was configured to format our PHP code. This next step applies the use of PHP CS Fixer tool to help with linting and any additional formatting not handled by Prettier.

### Relevant tickets

References [Pivotal #174412695](https://www.pivotaltracker.com/story/show/174412695).

### Checklist

- [x] This PR has been added to the relevant Pivotal card.
- [ ] Added appropriate feature/unit tests.

### CODE REVIEW

**Comment by weerd** [comment]
I was able to compare what conflicts exists between formatting from Prettier and the PHP CS Fixer and resolved any conflicts.

However, I plan on applying this process to Rogue as well since it is a larger code base and may showcase other conflicting rules I need to address. But this is a great start!

File: app/Http/Controllers/RedirectsController.php [hunk 1]
```diff
@@ -29,7 +29,7 @@ public function __construct(Fastly $fastly)
 
     /**
      * Display a listing of the resource.
-     * GET /redirects
+     * GET /redirects.
```
Comment (mendelB) [inline app/Http/Controllers/RedirectsController.php:32]: Haha I don't know how I feel about these but I understand.

Comment (weerd) [reply app/Http/Controllers/RedirectsController.php:32]: 😂 

> In formatting tools we trust.™

Comment (weerd) [reply app/Http/Controllers/RedirectsController.php:32]: It might be that there's a better way to docblock the endpoint? Not sure on that.

**Pull request merged**